### PR TITLE
Use raw strings in tests

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: src/Analyzers/CSharp/Tests/ConvertNamespace/ConvertToBlockScopedNamespaceAnalyzerTests.cs [hunk 1]
```diff
@@ -24,7 +24,9 @@ public static IEnumerable<object[]> EndOfDocumentSequences
             get
             {
                 yield return new object[] { "" };
-                yield return new object[] { "\r\n" };
+                yield return new object[] { """
+
+                    """ };
```
Comment (DoctorKrolic) [inline src/Analyzers/CSharp/Tests/ConvertNamespace/ConvertToBlockScopedNamespaceAnalyzerTests.cs:29]: I don't think this is intentional

File: src/Analyzers/CSharp/Tests/AddParameter/AddParameterTests.cs [hunk 2]
```diff
@@ -3030,13 +3032,15 @@ record struct Test(string V);
...
-struct Test();
-", @"new Test(""repro"");
+                struct Test();
+                """, """
+                new Test("repro");
 
-struct Test(string v);
-");
+                struct Test(string v);
+                """);
```
Comment (sharwell) [inline src/Analyzers/CSharp/Tests/AddParameter/AddParameterTests.cs:3043]: This pull request removes the first/final newline from inputs, which causes most tests to no longer represent the majority case that occurs in real code. It also makes it impossible to review this to try and identify cases that do/do not matter for the change in input semantics.

Comment (CyrusNajmabadi) [reply src/Analyzers/CSharp/Tests/AddParameter/AddParameterTests.cs:3043]: Honestly, it is just not that relevant.

Comment (ToddGrun) [reply src/Analyzers/CSharp/Tests/AddParameter/AddParameterTests.cs:3043]: Talked with Cyrus offline and he's verified that the input containing a newline at beginning/end isn't relevant to any of these tests. Other than that, this looks good to me and I'll go ahead and sign off.

**Review by ToddGrun — APPROVED** [review]
:shipit:

**Pull request merged**